fractions/javaee/jaxrs/src/main/java/org/wildfly/swarm/jaxrs/runtime/DefaultApplicationDeploymentProcessor.java

```diff
@@ -76,7 +76,9 @@ public void process() throws Exception {
             return;
         }
 
-        addGeneratedApplication(archive.as(JAXRSArchive.class));
+        if (applicationPath.isExplicit()) {
```

Comment:
Should probably change a bit the `@AttributeDocumentation` of `applicationPath`, to reflect the change.

Reply:
Let me take a stab at providing some useful documentation for the annotation. Will need a customer focussed doc-review of that line though.

Reply:
It's a single sentence, not an overly big deal. My suggestion: `If set, WildFly Swarm will automatically generate a JAX-RS Application class and use this value as the @ApplicationPath`

Reply:
Since I haven't looked at the existing documentation that much, I wasn't aware of the tone / technical details that are included in the documentation. Will use the above sentence and update.